spec/support/vcr_cassettes/mobile/appointments/VAOS_v2/get_appointments_with_mixed_provider_types.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
   - request:
       method: get
-      uri: https://veteran.apps.va.gov/vaos/v1/patients/1012846043V576341/appointments?end=2023-01-26T23:59:59Z&pageSize=0&start=2021-01-01T00:00:00Z&statuses=proposed,cancelled,booked,fulfilled,arrived
+      uri: https://veteran.apps.va.gov/vaos/v1/patients/1012846043V576341/appointments?end=<%= end_date || "2023-01-26T23:59:59Z" %>&pageSize=0&start=<%= start_date || "2021-01-01T00:00:00Z" %>&statuses=proposed,cancelled,booked,fulfilled,arrived
```

Comment:
I added erb tags with a default value because I wanted to do a test in which some results are in the past and some in the future. That's not really possible with the current test data and I didn't want to dummy up new test data for this. It's not simple use new cassettes from the vaos repo because we make more calls to fetch additional data than they do. The defaults aren't really necessary, especially since this is only called from one place, but I felt more comfortable leaving the original values. 

This is not an elegant solution, especially because you still have to provide the erb tags in the specs (although you can leave the values nil if you want). I think a better solution would be a ticket for updating our specs to use newer cassettes. One thing the vaos team does is put all the requests needed for a call in one cassette. So that includes facilities, etc. Could be worth thinking about.